pagopa/io-dev-api-server
# chore: [IOBP-406] adapted new wallet details apis

## Short description
This PR adapts some new wallet details informations and apis in order to be compliant at the new payment gateway

## List of changes proposed in this pull request
- Updated the swagger definition;
- Added an handling into `wallet_onboarding.html` that creates a mock payment method based on the user selection;
- Adapted the `generateUserWallet` method in order to have all the required information requested by the new wallet payment apis;
- Added a new `getWalletTypeFromPaymentMethodId` method that will get the wallet type from the payment method id selected

## How to test
Checkout this PR and test it with the following PR: https://github.com/pagopa/io-app/pull/5189 - You should be able to onboard all the available payment methods (in this case 2: CARD and PAYPAL)
